contracts/L2/token/LivepeerToken.sol

```diff
@@ -35,4 +35,13 @@ contract LivepeerToken is ILivepeerToken, AccessControl, ERC20Permit {
         _burn(_msgSender(), _amount);
         emit Burn(_msgSender(), _amount);
     }
+
+    function gatewayBurn(address _from, uint256 _amount)
```

Comment:
Since we're planning on disabling permissionless burning on L2 (in order to avoid dealing with the scenario where you burn LPT on L2 outside of bridging which poses the question of whether you should send a message to L1 to burn LPT in escrow there), WDYT about just modifying the existing `burn()` function with this function signature and so that it is only callable by addresses with the `BURNER_ROLE` and give the L2LPTGateway that role?

Reply:
flagging that L2LPTGateway (or any address with BURNER_ROLE) will have the ability to burn any address' LPT without their consent (also the case with gatewayBurn).